### DISCUSSION TITLE
Does cCart have good product filter options?

### DISCUSSION THREAD

**Discussion by dnationsoft12345** [imranertaza/cCart#147] [Q&A] [answered]
Hi
I need to know if there is an option that can filter products properly when a customer purchase any products.

Please let me know.
Thank you

**Comment by imranertaza** [comment, marked as answer]
Hi
Yes, it has a great feature that can help customer to filter different brands, category,  options like size and colors etc before he purchase any products.

We have developed this kind of feature in it so that the person who are going to use cCart, they can get good feedback from their customers for this good user experience.
Thank you
